examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/screens/paywallfooter/PaywallFooterScreen.kt

```diff
@@ -0,0 +1,96 @@
+package com.revenuecat.paywallstester.ui.screens.paywallfooter
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.revenuecat.paywallstester.ui.screens.paywall.PaywallScreenState
+import com.revenuecat.paywallstester.ui.screens.paywall.PaywallScreenViewModel
+import com.revenuecat.paywallstester.ui.screens.paywall.PaywallScreenViewModelImpl
+import com.revenuecat.purchases.ui.revenuecatui.PaywallFooter
+import com.revenuecat.purchases.ui.revenuecatui.PaywallViewOptions
+
+@Composable
+fun PaywallFooterScreen(
+    viewModel: PaywallScreenViewModel = viewModel<PaywallScreenViewModelImpl>(),
```

Comment:
I'm reusing the same view model as the full screen one. No need to create a separate view model.

Reply:
Yup 👍🏻